dataset_preparation/prepare_yfcc100m.py

```diff
@@ -46,10 +88,23 @@
     codec.train(xt)
     all_descriptors = mat
     all_descriptors_valid = np.load("/scratch/matthijs/concatenated_valid.npy")
```

Comment:
Shall we remove the parts that relate to your own machine Matthijs? Or is this just meant as an archival of how you prepared the dataset?

Reply:
@maumueller  @mdouze  I am merging this in the interest of time. We can follow up with another commit to clean up as necessary?

Reply:
yeah, I'd rather keep it around, so that I remember how I did it when we'll do bigann 2025 ;-)